glean-core/preview/src/lib.rs

```diff
@@ -207,5 +207,23 @@ pub fn submit_ping_by_name(ping: &str, reason: Option<&str>) -> bool {
     with_glean(|glean| glean.submit_ping_by_name(ping, reason).unwrap_or(false))
 }
 
+/// Add a debug tag to pings.
+///
+/// See `glean_core::Glean.set_debug_view_tag`.
+///
+/// ## Return value
+///
+/// Returns true if the tag was set, false otherwise.
+pub fn set_debug_view_tag(value: &str) -> bool {
+    with_glean_mut(|glean| glean.set_debug_view_tag(value))
+}
+
+/// Remove the debug view tag.
+///
+/// See `glean_core::Glean.clear_debug_view_tag`.
+pub fn clear_debug_view_tag() {
```

Comment:
Do we really need a function to clear the tag? I supposed that since we have one to set the tag we must need a way to clear it, but turns out none of the other bindings actually have a way to clear the tag once it's set.

Reply:
This might be needed for fog. @badboy ?

In general, though, I'm not sure it makes sense to keep "preview" up to date. What's your take on it, @badboy ?

Reply:
I will remove this commit for the `preview` changes for now. It's easy enough to bring it back later for a future PR and it is very decoupled from this PR anyways.